### PR TITLE
[DOC] Update Object Detection tutorial

### PR DESCRIPTION
**Pull request by GalyaZalesskaya**
What was done:
- Updated object detection tutorials according to new CLI
- Slightly updated installation section, adding Troubleshooting section
- Checked that new pages don't have OTX abbreviation (instead of OpenVINO Training Extensions)

### CODE REVIEW

File: docs/source/guide/tutorials/base/how_to_train/detection.rst [hunk 1]
```diff
@@ -134,55 +134,112 @@ Training
...
+  If you want to update your current workspace by running ``otx build`` with other parameters, it's better to delete the original workplace before that to prevent mistakes.
+
+Check ``otx-workspace-DETECTION/data.yaml`` to ensure, which data subsets will be used for training and validation, and update it if necessary.
+
+.. code-block::
+
+  data:
+  train:
+    ann-files: null
+    data-roots: /home/gzalessk/code/training_extensions/otx-workspace-DETECTION/splitted_dataset/train
```
Comment (kprokofi) [inline docs/source/guide/tutorials/base/how_to_train/detection.rst]: Please, delete home name + I think "ann-files" isn't presented in data.yaml, is it?

**Review by kprokofi — COMMENTED** [review]
Overall looks good to me, but, please, delete hardcoded paths and take a look into ann-files whether they exist or not

**Comment by harimkang** [comment]
Isn't this a release target PR?

File: docs/source/guide/tutorials/base/how_to_train/detection.rst (hunk 1 above)
Comment (GalyaZalesskaya) [reply docs/source/guide/tutorials/base/how_to_train/detection.rst]: Fixed. I copied this representation from data.yaml, so for object detection it's presented, although it's null

File: docs/source/guide/tutorials/base/how_to_train/detection.rst [hunk 2]
```diff
@@ -78,44 +77,45 @@ It's a great example to start with. The model achieves high accuracy right from
...
+      ├── val
+      └── test
+
+We can do that by running these commands:
 
+.. code-block::
 
-3. ``(Optional)`` To simplify the command line functions calling, we may create a ``data.yaml`` file with annotations info and pass it as a ``--data`` parameter. The content of the ``training_extesions/data.yaml`` for WGISD dataset should have absolute paths and will be similar to that:
+  # remove extra folders
+  rm -rf contrib/ extra/ original_resolution/
```
Comment (harimkang) [inline docs/source/guide/tutorials/base/how_to_train/detection.rst]: Do we need to do this?

**Review by harimkang — COMMENTED** [review]
I've left a comment about the lack of explanation from the user's point of view, and it seems like it could use a little more explanation if it's absolutely necessary code.

File: docs/source/guide/tutorials/base/how_to_train/detection.rst (hunk 2 above)
Comment (GalyaZalesskaya) [reply docs/source/guide/tutorials/base/how_to_train/detection.rst]: It was more for the prettiness of the result folder, but yes, it's not necessary. I removed it together with other extra code and added more explanation. Please tell me if I need to elaborate it more

**Review by harimkang — APPROVED** [review]

**Review by goodsong81 — APPROVED** [review]

**Pull request merged**